realtime_updater.py

```
@@ -0,0 +1,119 @@
+    )
+    conn.commit()
+    conn.close()
+
+
+def fetch_daily_stats(date: str) -> List[Dict[str, str]]:
+    """Fetch stats for a given date (YYYY-MM-DD)."""
+    try:
+        resp = requests.get(SCORES_API, params={"date": date}, timeout=10)
+        resp.raise_for_status()
+        return resp.json()
+    except Exception as exc:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)